src/main/java/de/neuland/jade4j/JadeConfiguration.java

```diff
@@ -44,22 +45,30 @@ public JadeConfiguration() {
             MAX_ENTRIES + 1).build();
 
     public JadeTemplate getTemplate(String name) throws IOException, JadeException {
+        return getTemplate(name, null, Collections.EMPTY_MAP);
+    }
+
+    public JadeTemplate getTemplate(String name, String languageCode, Map<String, String> translation) throws IOException, JadeException {
```

Comment:
also, related to comment below, `translation` can be renamed to better explain what the map is. e.g. `translationsByLanguage`